File: enemyShip.js
class EnemyShip {
    constructor(x, y) {
        this.x = x;
        this.y = y;
    }

    moveRandomly() {
        const directions = ['up', 'down', 'left', 'right'];
        const direction = directions[Math.floor(Math.random() * directions.length)];
        switch (direction) {
            case 'up':
                this.y -= 1;
                break;
            case 'down':
                this.y += 1;
                break;
            case 'left':
                this.x -= 1;
                break;
            case 'right':
                this.x += 1;
                break;
        }
    }

    draw(ctx) {
        ctx.fillStyle = 'red';
        ctx.fillRect(this.x * 20, this.y * 20, 20, 20);
    }
}

File: old/enemyShip.py
class EnemyShip:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def moveRandomly(self):
        import random
        directions = ['up', 'down', 'left', 'right']
        direction = random.choice(directions)
        if direction == 'up':
            self.y -= 1
        elif direction == 'down':
            self.y += 1
        elif direction == 'left':
            self.x -= 1
        elif direction == 'right':
            self.x += 1

    def draw(self, ctx):
        ctx.fillStyle = 'red'
        ctx.fillRect(self.x * 20, self.y * 20, 20, 20)
